irohad/main/iroha_conf_loader.hpp

```diff
@@ -103,6 +104,8 @@ inline rapidjson::Document parse_iroha_config(const std::string &conf_path) {
                    ac::no_member_error(mbr::MstSupport));
   ac::assert_fatal(doc[mbr::MstSupport].IsBool(),
                    ac::type_error(mbr::MstSupport, kBoolType));
+  ac::assert_fatal(doc[mbr::MstExpirationTime].IsUint(),
```

Comment:
Doesn't have to be fatal

Reply:
Not agreed - the type check has to be fatal, but the issue here is that the presence of this key - value pair is not checked and hence reported correctly. We should either treat it as optional and then check the type only when it is set, or demand its presence explicitly and fail with a corresponding message when it is not set.

Reply:
IMO should not be fatal, instead - some default value should be used